sql/sqltool/tool.go

```diff
@@ -311,12 +312,16 @@ func NewFlywayDir(path string) (*FlywayDir, error) {
 // and filename.
 func (d *FlywayDir) Files() ([]migrate.File, error) {
 	var ff flywayFiles
-	if err := fs.WalkDir(d, "", func(path string, e fs.DirEntry, err error) error {
+	if err := fs.WalkDir(d, ".", func(path string, e fs.DirEntry, err error) error {
 		if err != nil {
 			return err
 		}
-		if path != "" && e.IsDir() {
-			h, err := hidden(filepath.Join(d.Path(), path))
+		if path != "." && e.IsDir() {
+			fullPath := path
+			if d, ok := d.FS.(*migrate.LocalDir); ok {
+				fullPath = filepath.Join(d.Path(), path)
+			}
+			h, err := hidden(fullPath)
```

Comment:
```suggestion
			fullPath := path
			if l, ok := d.FS.(*migrate.LocalDir); ok {
				fullPath = filepath.Join(l.Path(), path)
			}
			h, err := hidden(fullPath)
```
Don't shadow the receiver